executor/reload_expr_pushdown_blacklist.go

```diff
@@ -36,18 +37,33 @@ func (e *ReloadExprPushdownBlacklistExec) Next(ctx context.Context, _ *chunk.Chu
 
 // LoadExprPushdownBlacklist loads the latest data from table mysql.expr_pushdown_blacklist.
 func LoadExprPushdownBlacklist(ctx sessionctx.Context) (err error) {
-	sql := "select HIGH_PRIORITY name from mysql.expr_pushdown_blacklist"
+	sql := "select HIGH_PRIORITY name, store_type from mysql.expr_pushdown_blacklist"
 	rows, _, err := ctx.(sqlexec.RestrictedSQLExecutor).ExecRestrictedSQL(sql)
 	if err != nil {
 		return err
 	}
-	newBlacklist := make(map[string]struct{}, len(rows))
+	newBlacklist := make(map[string]uint32, len(rows))
 	for _, row := range rows {
 		name := strings.ToLower(row.GetString(0))
+		storeTypeString := strings.ToLower(row.GetString(1))
 		if alias, ok := funcName2Alias[name]; ok {
 			name = alias
 		}
-		newBlacklist[name] = struct{}{}
+		var value uint32 = 0
+		if val, ok := newBlacklist[name]; ok {
+			value = val
+		}
+		storeTypes := strings.Split(storeTypeString, ",")
+		for _, typeString := range storeTypes {
+			if typeString == kv.TiDB.Name() {
```

Comment:
what if inputing wrong name, mixing capital and small letter like Tidb, tiFLASH, tikk?

Reply:
The input is case insensitive, but for wrong name, it will be ignored.